lib/travis/build/script/node_js.rb

```diff
@@ -4,7 +4,18 @@ module Travis
   module Build
     class Script
       class NodeJs < Script
-        DEFAULT_VERSION = '0.10'
+        DEFAULTS = {
+          node_js: '0.10',
+        }
+
+        DEPRECATIONS = [
+          {
+            name: 'Node.js',
+            current_default: '0.10',
+            new_default: 'lts/*',
+            cutoff_date: '2019-09-01',
```

Comment:
It’d be great if you could delay this, pending `lts/-2` being available?

Reply:
Sure. This date is completely arbitrary.